From 71d3217a16c4c029da31017378069d8ce65a19b3 Mon Sep 17 00:00:00 2001
From: Daniel Kurtz <djkurtz@chromium.org>
Date: Fri, 6 Mar 2015 17:11:56 +0800
Subject: [PATCH 2/2] tests: add rockchip to modetest, kmstest, vbltest and
 proptest

There is a rockchip drm kms driver.
Add "rockchip" to the static lists of driver names in the the standard
set of tests.

Signed-off-by: Daniel Kurtz <djkurtz@chromium.org>
---
 tests/kmstest/main.c      | 1 +
 tests/modetest/modetest.c | 2 +-
 tests/proptest/proptest.c | 2 +-
 tests/vbltest/vbltest.c   | 2 +-
 4 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/tests/kmstest/main.c b/tests/kmstest/main.c
index 449d75f..3f9a292 100644
--- a/tests/kmstest/main.c
+++ b/tests/kmstest/main.c
@@ -62,6 +62,7 @@ char *drivers[] = {
 	"nouveau",
 	"vmwgfx",
 	"exynos",
+	"rockchip",
 	NULL
 };
 
diff --git a/tests/modetest/modetest.c b/tests/modetest/modetest.c
index 4b9cf2f..8949cef 100644
--- a/tests/modetest/modetest.c
+++ b/tests/modetest/modetest.c
@@ -1453,7 +1453,7 @@ int main(int argc, char **argv)
 	int drop_master = 0;
 	int test_vsync = 0;
 	int test_cursor = 0;
-	const char *modules[] = { "i915", "radeon", "nouveau", "vmwgfx", "omapdrm", "exynos", "tilcdc", "msm", "sti", "tegra" };
+	const char *modules[] = { "i915", "radeon", "nouveau", "vmwgfx", "omapdrm", "exynos", "tilcdc", "msm", "sti", "tegra", "rockchip" };
 	char *device = NULL;
 	char *module = NULL;
 	unsigned int i;
diff --git a/tests/proptest/proptest.c b/tests/proptest/proptest.c
index 7618f63..a6011bf 100644
--- a/tests/proptest/proptest.c
+++ b/tests/proptest/proptest.c
@@ -303,7 +303,7 @@ static void printUsage(void)
 
 int main(int argc, char *argv[])
 {
-	char *modules[] = { "i915", "radeon", "nouveau", "vmwgfx", "omapdrm", "msm" };
+	char *modules[] = { "i915", "radeon", "nouveau", "vmwgfx", "omapdrm", "msm", "rockchip" };
 	unsigned int i, ret = 0;
 
 	for (i = 0; i < ARRAY_SIZE(modules); i++){
diff --git a/tests/vbltest/vbltest.c b/tests/vbltest/vbltest.c
index cdc1ef6..605cfd2 100644
--- a/tests/vbltest/vbltest.c
+++ b/tests/vbltest/vbltest.c
@@ -105,7 +105,7 @@ static void usage(char *name)
 int main(int argc, char **argv)
 {
 	int i, c, fd, ret;
-	char *modules[] = { "i915", "radeon", "nouveau", "vmwgfx", "exynos", "omapdrm", "tilcdc", "msm", "tegra" };
+	char *modules[] = { "i915", "radeon", "nouveau", "vmwgfx", "exynos", "omapdrm", "tilcdc", "msm", "tegra", "rockchip" };
 	drmVBlank vbl;
 	drmEventContext evctx;
 	struct vbl_info handler_info;
-- 
2.2.0.rc0.207.ga3a616c

